### PR TITLE
Update launch screen to look like blank app loading

### PR DESCRIPTION
**Pull request by kristen**
#24

### CODE REVIEW

File: Mapstagram/Base.lproj/LaunchScreen.xib [hunk 1]
```diff
@@ -11,27 +10,29 @@
...
-                </label>
-                <label opaque="NO" clipsSubviews="YES" userInteractionEnabled="NO" contentMode="left" horizontalHuggingPriority="251" verticalHuggingPriority="251" text="Mapstagram" textAlignment="center" lineBreakMode="middleTruncation" baselineAdjustment="alignBaselines" minimumFontSize="18" translatesAutoresizingMaskIntoConstraints="NO" id="kId-c2-rCX">
-                    <rect key="frame" x="20" y="140" width="441" height="43"/>
-                    <fontDescription key="fontDescription" type="boldSystem" pointSize="36"/>
-                    <color key="textColor" cocoaTouchSystemColor="darkTextColor"/>
-                    <nil key="highlightedColor"/>
-                </label>
+                <view contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="g4d-C2-rLm">
+                    <rect key="frame" x="0.0" y="0.0" width="480" height="64"/>
+                    <color key="backgroundColor" red="0.0" green="0.74901960779999999" blue="0.56078431370000004" alpha="1" colorSpace="calibratedRGB"/>
```
Comment (kristen) [inline Mapstagram/Base.lproj/LaunchScreen.xib]: I hard coded the color here. Not sure if we can do it elsewhere...

Comment (jamietsao) [inline Mapstagram/Base.lproj/LaunchScreen.xib]: I googled this yesterday, and it doesn't seem possible to change anything about the launch screen programmatically.  Not sure.  I think this is fine.

**Comment by jamietsao** [comment]
Hard to tell what this looks like by just reviewing this so merge it!  Excited to see this!

**Comment by kristen** [comment]
All I did was make it look like there is a nav bar and a tab bar. We can improve it later, I just wanted to put something out there quickly.

**Pull request merged**